### PR TITLE
DOCS-5584 preview and publish community

### PR DESCRIPTION
**Pull request by wwarner-salesforce**
DOCS-5584 preview and publish community

### CODE REVIEW

**Comment by wwarner-salesforce** [comment]
Based on review in https://github.com/mulesoft/docs-api-community-manager/pull/64

File: modules/ROOT/pages/preview-community.adoc [hunk 1]
```diff
@@ -4,10 +4,9 @@ include::_attributes.adoc[]
 endif::[]
```
Comment (kbostrom) [inline modules/ROOT/pages/preview-community.adoc:4]: NOTE
Very quick copyedit of changed material only

File: modules/ROOT/pages/preview-community.adoc [hunk 2]
```diff
@@ -4,10 +4,9 @@ include::_attributes.adoc[]
...
 :imagesdir: ../assets/images/
 
-To see a live preview of your community's site, including APIs only visible to the administrators, open the ACM Control Panel and select *Community Builder*.
+After you have edited your community site, you can preview the new version of the site to ensure that it meets your expectations:
```
Comment (kbostrom) [inline modules/ROOT/pages/preview-community.adoc]:
```suggestion
After you edit your community site, you can preview the new version to ensure that it meets your expectations:
```

File: modules/ROOT/pages/preview-community.adoc [hunk 3]
```diff
@@ -4,10 +4,9 @@ include::_attributes.adoc[]
...
 
-image::builder.png[Site Builder]
-
-Select *Preview*.
-
-To return to the builder, select *Back to Builder*.
+. From the API Community Manager (ACM) Control Panel, click *Community Builder*.
```
Comment (kbostrom) [inline modules/ROOT/pages/preview-community.adoc]:
```suggestion
. From the API Community Manager (ACM) control panel, click *Community Builder*.
```

File: modules/ROOT/pages/preview-community.adoc [hunk 4]
```diff
@@ -4,10 +4,9 @@ include::_attributes.adoc[]
...
+. Click *Preview*.
+. Examine your changes and verify that they appear correctly. APIs that are only visible to administrators will appear in this preview.
```
Comment (kbostrom) [inline modules/ROOT/pages/preview-community.adoc]:
```suggestion
. Verify that your changes are applied and display correctly. APIs that are visible only to administrators appear in this preview.
```

Comment (kbostrom) [reply modules/ROOT/pages/preview-community.adoc]: ...but is "only" meant to modify "APIs": Only APIs that are visible to administrators...?

File: modules/ROOT/pages/publish-community.adoc [hunk 5]
```diff
@@ -4,12 +4,18 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-To make your community site live, open the ACM Control Panel and select *Community Builder*.
+After you have edited and previewed your community site, you can publish the new version for your community users:
```
Comment (kbostrom) [inline modules/ROOT/pages/publish-community.adoc]:
```suggestion
After you edit and preview your updated community site, you can publish the new version for your users:
```

File: modules/ROOT/pages/publish-community.adoc [hunk 6]
```diff
@@ -4,12 +4,18 @@ include::_attributes.adoc[]
...
 
-image::builder.png[Site Builder]
+. From the API Community Manager (ACM) Control Panel, click *Community Builder*.
```
Comment (kbostrom) [inline modules/ROOT/pages/publish-community.adoc]:
```suggestion
. From the API Community Manager (ACM) control panel, click *Community Builder*.
```

File: modules/ROOT/pages/publish-community.adoc [hunk 7]
```diff
@@ -4,12 +4,18 @@ include::_attributes.adoc[]
...
+. Click *Publish*.
 
-Select *Publish*.
+If your community is not already active, activate it:
 
-To make your community active, open the ACM Control Panel, select *Community Administration* and *Settings*, and select *Activate*.
+. From the API Community Manager (ACM) Control Panel, click *Community Administration*.
```
Comment (kbostrom) [inline modules/ROOT/pages/publish-community.adoc]:
```suggestion
. From the API Community Manager (ACM) control panel, click *Community Administration*.
```

File: modules/ROOT/pages/publish-community.adoc [hunk 8]
```diff
@@ -4,12 +4,18 @@ include::_attributes.adoc[]
...
+. Click *Settings*.
+. Click *Activate*.
 
-To view the live site when the community is published and active, open the ACM Control Panel and select *Open Community*.
+After your community is published and active, view the live site:
+
+. Open the API Community Manager (ACM) Control Panel.
```
Comment (kbostrom) [inline modules/ROOT/pages/publish-community.adoc]:
```suggestion
. Open the API Community Manager (ACM) control panel.
```

**Review by kbostrom — APPROVED** [review]
DONE!

File: modules/ROOT/pages/preview-community.adoc (hunk 4 above)
Comment (wwarner-salesforce) [reply modules/ROOT/pages/preview-community.adoc]: Your change is exactly correct. APIs that are visible only to admins appear in this preview.

Comment (wwarner-salesforce) [reply modules/ROOT/pages/preview-community.adoc]: And other APIs also appear in this preview.

**Pull request merged**